src/state/gameSlice.ts

```diff
@@ -4,9 +4,13 @@ export interface GameState {
   gameCode?: string;
   gameCreationCallTs?: number;
   hasJoinedGame?: boolean;
+  gameStarted?: boolean;
```

Comment:
These 2 that you added don't need the question mark here (since that is just for things which must be allowed to be undefined).`gameStarted` can be set to false by default in GameState. That way no hooks register a change if `gameStarted` changes from undefined to false (it should be false in the first place)